catalog/csr-gitops/cloudbuild-iam.yaml

```diff
@@ -2,30 +2,32 @@
 apiVersion: iam.cnrm.cloud.google.com/v1beta1
 kind: IAMPolicyMember
 metadata:
-  name: deployment-repo-cloudbuild-write
-  namespace: config-control # {"$kpt-set":"namespace"}
   annotations:
+    cnrm.cloud.google.com/blueprint: cnrm/csr-gitops/v0.2.0
```

Comment:
Wondering why 0.2.0 as opposed to 0.1.0?

Reply:
v0.1.0 was already released: https://github.com/GoogleCloudPlatform/blueprints/tree/v0.1.0

We're now working towards v0.2.0: https://github.com/GoogleCloudPlatform/blueprints/tree/v0.2.0-alpha.1